packages/wrangler/lib/functions/buildWorker.ts

```diff
@@ -0,0 +1,42 @@
+import path from "path";
+import { build } from "esbuild";
+import { fileURLToPath } from "url";
+
+let sanitizedDirname;
+try {
+  sanitizedDirname = path.dirname(fileURLToPath(import.meta.url));
+} catch {
+  sanitizedDirname = __dirname;
+}
```

Comment:
what does this code do? (I don't know.) we're compiling the code so we may not need import.meta.url.

Reply:
I'm not 100% certain on it myself. I had a look, at it seemed to be primarily for Windows compatibility. `fileURLToPath` normalizes `C:\Users\gregbrimble\projects` to Node-land's `/Users/brimble/projects`.

I had to add the `catch` fallback to make the tests pass because we've got some trickery with `import.meta.url`.

Again, I plan to make improvements here incrementally after we get this out the door.

Reply:
Sure, yeah, switching it over to just use `__dirname`.